src/components/panel/panelSections/panelContent.component.tsx

```diff
@@ -105,76 +114,97 @@ export const PanelContent = ({
     onReorder(updatedObjects);
   };
 
+  if (draggedObject)
+    return (
+      <div
+        ref={setNodeRef}
+        className={clsx(
+          isOver && "border-primary text-primary",
+          "m-4 mt-10 flex h-72 items-center justify-center border-2 border-dotted text-center text-manatee-400",
+        )}
+      >
+        <span>{"Drop object here to add to the Set's content"}</span>
```

Comment:
Don't hardcode the Object Type. `Drop object here to add to the Object's content` (or pass `Object Type` down